### PR TITLE
Fix ToSvgString

### PR DESCRIPTION
**Pull request by wieslawsoltes**
#### What does this implement/fix? Explain your changes.

The `ToSvgString` method used current culture which produces invalid path data on some culture (uses `,` instead of `.` for separator.

### CODE REVIEW

File: Source/Paths/SvgPathBuilder.cs [hunk 1]
```diff
@@ -14,7 +14,7 @@ public static class PointFExtensions
     {
         public static string ToSvgString(this PointF p)
         {
-            return p.X.ToString() + " " + p.Y.ToString();
+            return p.X.ToString(CultureInfo.InvariantCulture) + " " + p.Y.ToString(CultureInfo.InvariantCulture);
```
Comment (mrbean-bremen) [inline Source/Paths/SvgPathBuilder.cs]: Thanks - not the first time I see such an error!

**Comment by H1Gdev** [comment]
@wieslawsoltes 
@mrbean-bremen 

https://github.com/vvvv/SVG/blob/a6515547b5e49ee6844611d0423155c24199d434/Source/Paths/SvgPathBuilder.cs#L485-L496

I don't know which of way is good, but I think that it is better to unify either.

**Comment by wieslawsoltes** [comment]
Some example:

https://github.com/dotnet/wpf/blob/9f9c0426b33dee76319460e56f9f0459aee22206/src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Media/LineSegment.cs#L107

https://github.com/dotnet/wpf/blob/52f24ad1517fc4412c989ce684189d7249abd137/src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Media/Converters/Generated/PathFigureCollectionValueSerializer.cs#L108

**Comment by wieslawsoltes** [comment]
Seems like `SvgPathBuilder` does not need to change culture after my latest PRs.

**Comment by mrbean-bremen** [comment]
I would prefer to always use the explicit invariant culture for `float.ToString()` and `float.TryParse()`, where the representation is fixed (as it is in SVG) - this is less error-prone IMHO. 
So I like it the way @wieslawsoltes has done this, and I agree that the code in `SvgPathBuilder` can probably be removed (maybe can be added to the other PR).

**Pull request merged**